File: typescript/src/server/address/lookup/restApiAddressLookupV2Response.ts
import { RestApiResponse } from "../../restApiResponse";
import { RestApiResponseError } from "../../restApiResponseError";
import { RestApiAddressLookupV2Result } from "./restApiAddressLookupV2Result ";

export type RestApiAddressLookupV2Response = RestApiResponse &  {
    error?: RestApiResponseError;
    result?: RestApiAddressLookupV2Result
}

File: csharp/DVSClient/DVSClient/server/address/lookup/RestApiAddressLookupV2Response.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Lookup
{
    public class RestApiAddressLookupV2Response : RestApiResponse
    {
        [JsonProperty("error")]
        public RestApiResponseError? Error { get; set; }
        [JsonProperty("result")]
        public RestApiAddressLookupV2Result? Result { get; set; }
    }
}